app/js/components/identifiers/identifiers.jsx

```diff
@@ -0,0 +1,379 @@
+/* * This Source Code Form is subject to the terms of the Mozilla Public License,
+ * v. 2.0. If a copy of the MPL was not distributed with this file, You can
+ * obtain one at http://mozilla.org/MPL/2.0/. OpenMRS is also distributed under
+ * the terms of the Healthcare Disclaimer located at http://openmrs.org/license.
+ *
+ * Copyright (C) OpenMRS Inc. OpenMRS is a registered trademark and the OpenMRS
+ * graphic logo is a trademark of OpenMRS Inc.
+ */
+import React from 'react';
+import apiCall from '../../utilities/apiHelper';
+import NewIdentifier from './addIdentifier';
+
+export default class Identifiers extends React.Component {
+    constructor(props) {
+        super(props);
+        this.state = {
+            patientUuid: props.uuid,
```

Comment:
Just as good practice how about you include a 
`  componentWillReceiveProps` method that checks to see if the uuid set to state is similar to one in props to ensure state does not misbehave when switching in between different patients.
See the sample code below:
`  componentWillReceiveProps(nextProps) {
    if (this.state.patientUuid !== this.props.uuid) {
      this.setState({
        patientUuid: this.props.uuid
      })

    }
  }`

Reply:
I do not think this is necessary considering the fact that the props(patientUuid) is passed once in the lifecycle of this component and never changed through out. We are not using Redux/Flux where the state could change using stores. From the code, parentUuid was never changed so I think adding componentWillReceiveProps just to check patientUuid will not be necessary.

Reply:
noted. Will do. Thanks.

Reply:
@CeciliaWanjiku What I think could be done is: do not set it as a state.

Reply:
I see what you mean @andela-cuchendu about setting it to state.
Thanks.

Reply:
@CeciliaWanjiku there is no need compare the state with a value it already contains. When a `prop` is passed to a component, the prop does not change for any reason except if explicitly changed.
So updating the `state` with a value that it contains which hasn't changed is redundant and expensive. Remember that whenever `setState` is called the component is re-rendered, so you might want to use it only when needed.